sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp

```diff
@@ -40,6 +40,7 @@
 #include <string>
 #include <thread>
 #include <utility>
+#include <iostream>
```

Comment:
and this too

Reply:
I can't remove that for some reason. Otherwise the build fails , even though there's no reference to any <iostream> object within the file